packages/serverpod/lib/src/server/serverpod.dart

```diff
@@ -22,6 +22,9 @@ import '../cache/caches.dart';
 import '../generated/endpoints.dart' as internal;
 import '../generated/protocol.dart' as internal;
 
+/// The default server id if none is specified on startup.
+const defaultServerId = 'default';
+
```

Comment:
Question: When reviewing commit by commit and looking at the second commit this doesn't seem to be used. Can it be removed?